docs/world api.md

```diff
@@ -202,46 +205,43 @@ class MyGameWorld(World):
+Items are all things that can "drop" for your game. This may be RPG items like weapons, or technologies you normally
+research in a research tree.
 
-Each item has a `name`, an `id` (can be known as "code"), and a classification.
-The most important classification is `progression` (formerly advancement).
-Progression items are items which a player may require to progress in
-their world. Progression items will be assigned to locations with higher
-priority and moved around to meet defined rules and accomplish progression
-balancing.
+Each item has a `name`, a `code` (hereafter referred to as `id`), and a classification.
+The most important classification is `progression`. Progression items are items which a player *may* require to progress
+in their world. If an item can possibly be considered for logic (it's referenced in a location's rules) it *must* be
+progression. Progression items will be assigned to locations with higher priority and moved around to meet defined rules
+and accomplish progression balancing.
 
-The name needs to be unique in each game, meaning a duplicate item has the
-same ID. Name must not be numeric (has to contain at least 1 letter or symbol).
+The name needs to be unique in each game, meaning a duplicate item has the same ID. Name must not be numeric (must
+contain at least 1 letter or symbol).
 
-Special items with ID `None` can mark events (read below).
+Other classifications include:
 
-Other classifications include
 * `filler`: a regular item or trash item
-* `useful`: generally quite useful, but not required for anything logical
+* `useful`: generally quite useful, but not required for anything logical. Cannot be placed on excluded locations
 * `trap`: negative impact on the player
 * `skip_balancing`: denotes that an item should not be moved to an earlier sphere for the purpose of balancing (to be
   combined with `progression`; see below)
 * `progression_skip_balancing`: the combination of `progression` and `skip_balancing`, i.e., a progression item that
-  will not be moved around by progression balancing; used, e.g., for currency or tokens
+  will not be moved around by progression balancing; used, e.g., for currency or tokens, to not flood early spheres
 
 #### Documenting Items
 
-Worlds can optionally provide an `item_descriptions` map which contains
-human-friendly descriptions of items or item groups. These descriptions will
-show up in item-selection options in the Weighted Options page. Extra
-indentation and single newlines will be collapsed into spaces.
+Worlds can optionally provide an `item_descriptions` map which contains human-friendly descriptions of items or item
+groups. These descriptions will show up in item-selection options in the Weighted Options page. Extra indentation and
+single newlines will be collapsed into spaces.
 
 ```python
-# Items.py
+# items.py
 
 item_descriptions = {
-    "Red Potion": "A standard health potion",
+    "Red Potion":    "A standard health potion",
```

Comment:
Again, the two extra spaces do not improve readability and black would remove them.